airbyte-integrations/connectors/source-stripe/unit_tests/integration/test_payment_methods.py

```diff
@@ -129,11 +124,23 @@ def test_given_one_page_when_read_then_return_records(self, http_mocker: HttpMoc
     @HttpMocker()
     def test_given_many_pages_when_read_then_return_records(self, http_mocker: HttpMocker) -> None:
```

Comment:
Can we specify how many pages on what? It seems to be on the child for this case. Do we need a test that validates pagination on the parent?

Reply:
specified, set to 2

> Do we need a test that validates pagination on the parent?

we already have pagination for parent in `test_given_multiple_customers_pages_when_read_then_query_pagination_on_parent`